bootstrap/bootstrap-storage.sh

```diff
@@ -41,60 +40,57 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
+if grep -E "^(\s)*db.type:(\s)*\"mysql\"(\s)*$" $CONFIG_FILE_PATH > /dev/null; then
```

Comment:
Yeah, this will be error-prone. I will replace a subset of classes in tool to download MySQL dependencies. After that I can use StreamlineApplication class to trigger the migrations